convert2rhel/subscription.py

```diff
@@ -674,6 +671,16 @@ def attach_subscription():
     """
     # TODO: Support attaching multiple pool IDs.
 
+    # check if SCA is enabled
+    output, _ = utils.run_subprocess(["subscription-manager", "status"], print_output=False)
+    if "Content Access Mode is set to Simple Content Access." in output:
```

Comment:
I think we should compare the output here in lower case, right? In case that they change the casing in the future, it will still work as it will check for the lower-case version of that output always. 

```suggestion
    if "content access mode is set to simple content access." in output.lower():
```